src/components/Login.tsx

```diff
@@ -10,19 +10,39 @@ import {
   Input,
 } from '@chakra-ui/react';
 import { useForm } from 'react-hook-form';
+import { useRouter } from 'next/router';
 
 type LoginProps = {
-  onSubmit: () => void;
+  apiURL: string;
+  replaceURL: string;
   title: string;
 };
 
-const Login = ({ onSubmit, title }: LoginProps) => {
+const Login = ({ apiURL, replaceURL, title }: LoginProps) => {
+  const router = useRouter();
+
   const {
     handleSubmit,
     register,
     formState: { errors, isSubmitting },
+    getValues,
   } = useForm();
 
+  let inputs: any;
+
+  const onSubmit = () => {
+    fetch(apiURL, {
+      method: 'POST',
+      body: JSON.stringify(inputs),
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    })
+      .then(() => router.replace(replaceURL))
```

Comment:
I believe the Fetch API will resolve the promise successfully even if the response has an error status code (400-500s). So, you will need to do further checks in this promise handler to check if the login request was successful. Ref: https://www.tjvantoll.com/2015/09/13/fetch-and-errors/